spec/features/admin_adds_a_group_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+feature "Admin adds a group" do
+  scenario "successfully" do
+    visit new_group_path
+
+    fill_in "name", with: "St. Peter", :match => :prefer_exact
```

Comment:
Use:
1. field's name (i.e. `group[name]`)
2. field's id (i.e. `group_name`)
3. A translation for the field's label.

Which will then remove the need to use the `match: :prefer_exact`

Reply:
I couldn't make this work. So I used translation and I added this to `capybara_webkit.rb` : 

`Capybara.configure do |config|`
    `config.match = :prefer_exact`
`end`

Is it ok? Done in sha `fedba1c`

I also changed a little bit the structure, I added 2 variables for the dates, in fixup `55f851c`